src/ShopBundle/Interfaces/ShopServiceInterface.php

```diff
@@ -81,4 +81,9 @@ public function getActiveCategory();
     public function getActiveBasket();
 
     public function resetBasket();
+
+    /**
+     * @return null|\TdbShopManufacturer
+     */
+    public function getActiveManufacturer();
```

Comment:
Interface changes are not permitted in patch level releases

Reply:
ok, moved to a private method in the module